docker/Dockerfile

```diff
@@ -22,9 +23,9 @@ WORKDIR ${SITE_DIR}
 RUN install -d proj/ var/log/ htdocs/ htdocs/static/
 
 # Install python packages
-ADD requirements.txt ${SITE_DIR}requirements.txt
-ADD requirements_test.txt ${SITE_DIR}requirements_test.txt
-RUN pip install -r ${SITE_DIR}requirements_test.txt
+ADD Pipfile ${SITE_DIR}Pipfile
+ADD Pipfile.lock ${SITE_DIR}
```

Comment:
Quick question - Should the new line 27 be `Pipfile.lock ${SITE_DIR}Pipfile.lock`?  Looks odd compared to Pipfile and the old requirements.txt  and the requirements_test.txt.

Reply:
I'll fix this with a rebase